python_modules/libraries/dagster-dg/dagster_dg_tests/cli_tests/test_component_commands.py

```diff
@@ -147,31 +147,6 @@ def test_component_generate_already_exists_fails(in_deployment: bool) -> None:
 # ########################
 
 
-def test_generate_sling_replication_instance() -> None:
```

Comment:
Is there a reason this test is deleted and not updated to use `sling_replication_collection_component`?

Reply:
ah yeah the new collection component doesn't generate files anymore